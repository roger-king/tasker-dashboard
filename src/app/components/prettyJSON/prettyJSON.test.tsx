import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import PrettJSON from './index';

Enzyme.configure({ adapter: new Adapter() });

describe('<PrettyJSON />', () => {
    const data = {
        name: 'Steve Rogers',
        email: 'steve.rogers@avengers.com',
        friends: [
            {
                name: 'Tony Stark',
                alias: 'Iron Man',
            },
        ],
    };

    it('Matches the snapshot', () => {
        const prettyJSON = renderer.create(<PrettJSON data={data} />).toJSON();
        expect(prettyJSON).toMatchSnapshot();
    });
});
