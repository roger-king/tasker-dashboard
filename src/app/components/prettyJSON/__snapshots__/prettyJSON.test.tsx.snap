// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrettyJSON /> Matches the snapshot 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 kDYwmu sc-bdVaJa dIhcOP"
>
  <pre>
    {
    "name": "Steve Rogers",
    "email": "steve.rogers@avengers.com",
    "friends": [
        {
            "name": "Tony Stark",
            "alias": "Iron Man"
        }
    ]
}
  </pre>
</div>
`;
